packages/core/src/browser/quick-open/quick-command-service.ts

```diff
@@ -24,7 +24,8 @@ export class QuickCommandService implements QuickOpenModel {
     open(): void {
         // let's compute the items here to do it in the context of the currently activeElement
         this.items = [];
-        for (const command of this.commands.commands) {
+        const filteredAndSortedCommands = this.commands.commands.filter(a => a.label).sort((a, b) => a.label!.localeCompare(b.label!));
```

Comment:
Forgive my laziness but what is `label!` here ? 
Also the sorting is done alphabetically now is that better than the fuzzy sort ?

Reply:
I will rename it :)
Yes, it is better then fuzzy sort, which does sort based on the length of the label (and other unrelated things).

Reply:
Ah, you mean the exclamation mark? It tells typescript to not worry about `undefined`. It doesn't statically understand that I did filter already before.

Reply:
Yes... OK I had no idea thx.